app/javascript/leaflet/viewers/wms.js

```diff
@@ -0,0 +1,81 @@
+import { tileLayer } from "leaflet";
+import LeafletViewerMap from "./map.js";
+
+export default class LeafletViewerWms extends LeafletViewerMap {
+  onLoad() {
+    super.onLoad();
```

Comment:
This function has additional code not in the original but I am pretty sure that is fine.
```
if (this.map.getBounds().isValid()) {
      this.element.setAttribute(
        "data-js-map-render-bbox",
        this.map.getBounds().toBBoxString()
      );
    }
 ```